examples/resources/snowflake_connection/resource.tf

```diff
@@ -0,0 +1,37 @@
+## Minimal
+resource "snowflake_connection" "basic" {
+  name = "connection_name"
+}
+
+## Enable failover to accounts
+resource "snowflake_connection" "with_enable_failover_list" {
```

Comment:
Nit: I don't think we need this one. This additional field is already covered below.

Reply:
true